src/presenter/point-presenter.js

```diff
@@ -0,0 +1,112 @@
+import { render, replace } from '../framework/render.js';
+import EventsView from '../view/event-view.js';
+import EventEdit from '../view/event-edit.js';
+
+const PointMode = {
+  VIEW: 'view',
+  EDIT: 'edit',
+};
+
+export default class PointPresenter{
+
+  #eventsListView = null;
+  #point = null;
+  #offer = null;
+  #destination = null;
+  #points = null;
+
+  #pointComponent = null;
+  #pointEditComponent = null;
+  #openEditModeHandler = null;
+
+  #mode = null;
+  #modeFavorit = null;
+  #openFavoritModeHandler = null;
+  #changeFavorite = null;
+
+  constructor({eventsListView, point, offers, destinations}){
```

Comment:
Добавляем сразу `handleDataChange` можно и потом, но можно и тут, нужно всегда, можно и через конструктор поэтому 
```suggestion
  constructor({eventsListView, offersModel, destinationsModel, onDataChange}) {
    this.#eventsListView = eventsListView;
    this.#offersModel = offersModel;
    this.#destinationsModel = destinationsModel;
    this.#handleDataChange = onDataChange;
  }
```